develop-docs/application/dynamic-sampling/outcomes.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: The "indexed" outcome category
+sidebar_order: 4
+---
+
+Data types which are subject to dynamic sampling are represented in two different outcome data categories: "total" and "indexed". You can think of the "total" category as counting items that are stored in _aggregated_ form, i.e. as metrics. The "indexed" category counts items that are stored individually.
+
+If an item is dropped _before_ metrics extraction & dynamic sampling, it will show up in both categories (for example, `"transaction"`  and `"transaction_indexed"`). If an item is dropped _by_ dynamic sampling, it will cease to exist in the "indexed" category, but will live on in the form of metrics.
```

Comment:
It still exists in the indexed category just as a negative outcome `Filtered` (but the total category it can still be accepted).

Reply:
👍 I rephrased.